llvm/include/llvm/Frontend/OpenMP/OMPIRBuilder.h

```diff
@@ -2801,6 +2801,67 @@ class OpenMPIRBuilder {
   using GenMapInfoCallbackTy =
       function_ref<MapInfosTy &(InsertPointTy CodeGenIP)>;
 
+private:
+  /// Emit the array initialization or deletion portion for user-defined mapper
+  /// code generation. First, it evaluates whether an array section is mapped
+  /// and whether the \a MapType instructs to delete this section. If \a IsInit
+  /// is true, and \a MapType indicates to not delete this array, array
+  /// initialization code is generated. If \a IsInit is false, and \a MapType
+  /// indicates to not this array, array deletion code is generated.
+  void emitUDMapperArrayInitOrDel(Function *MapperFn, llvm::Value *Handle,
```

Comment:
Document Handle, e.g. add a \param for it.

Reply:
I've updated the name to MapperHandle to better describe the param. Do we need a /param description for it given that this is a private member function?